test/utils_tests.js

```diff
@@ -108,5 +108,72 @@ describe('Utils', function () {
             assert.equal(best[0].element.id, 'fat');
         });
 
+        it('test non-attribute', function () {
```

Comment:
I don't understand this test title. How is this "non-attribute"? It still seems to be using attributes, in the form of `rule(type('attr'), score(scoreFunc))`. Does "attribute" means something special in Fathom?

Reply:
Mm I was referring to a html element's attributes (e.g. id, class, etc.) rather than the type I've given it in fathom, and was trying to test the case when a user tries to search an attribute that the element does not have (non-attribute?).

Reply:
Oh, I see. The test here is that the first element has the attribute, and the second one doesn't, so the second doesn't get included in the results? A comment to that affect could be useful.